### DISCUSSION TITLE
On web3modal V2, How can I select a specific wallet?

### DISCUSSION THREAD

**Discussion by shenhaoguangDayDayUp** [WalletConnect/walletconnect-monorepo#1757] [Developer Support]
I don't want to display all of my wallets, and I want to put my designated wallet at the front.

**Comment by chadyj** [comment]
Take a look at the config options

https://docs.walletconnect.com/2.0/web3modal/configuration#mobilewallets-optional
https://docs.walletconnect.com/2.0/web3modal/configuration#desktopwallets-optional

**Comment by jan-miksik** [comment]
You may want something like this?
https://docs.walletconnect.com/2.0/web3modal/configuration#explorerallowlist-optional

```
const web3modal = new Web3Modal(
    {
      projectId: import.meta.env.VITE_WALLET_CONNECT_PROJECT_ID,
      explorerAllowList: [
        // metamask
        'c57ca95b47569778a828d19178114f4db188b89b763c899ba0be274e97267d96',
      ],
    },
    ethereumClient,
  )
```
